src/components/Chatting/MessageForm/FullChatMessageForm.tsx

```diff
@@ -51,16 +58,42 @@ const BtnImage = (props) => {
     </div>
   );
 };
-BtnImage.propTypes = {
-  onClick: PropTypes.func,
+
+type BtnAccountProps = {
+  onClick: () => void;
 };
 
-const FullChatMessageForm = (props) => {
-  const textareaContRef = useRef();
-  const textareaRef = useRef();
-  const inputImage = useRef();
+const BtnAccount = (props: BtnAccountProps) => {
+  const style = {
+    width: "22px",
+    minWidth: "22px",
+    height: "22px",
+    marginBottom: "5px",
+    ...theme.cursor(),
+  };
+  return (
+    <div style={style} onClick={props.onClick}>
+      <LocalAtmIcon
+        style={{ width: "100%", height: "100%", fill: theme.gray_text }}
+      />
+    </div>
+  );
+};
+
+type FullChatMessageFormProps = {
+  handleSendMessage: (message: string) => boolean;
+  handleSendImage: (image: File) => void;
+  handleSendAccount: (account: string) => void;
```

Comment:
위 리뷰랑 동일하게, 

```suggestion
  handleSendAccount: (account: string) => boolean;
```
이요